lib/compiler.js

```diff
@@ -22,33 +25,36 @@ Compiler.prototype.generate = function generate(code, options) {
 
   var locals = this.runtime.prototype.locals;
 
-  var source = [
-    '/// -------------------------------------',
-    '/// --------- BEM-XJST Runtime Start ----',
-    '/// -------------------------------------',
-    'var ' + exportName + ' = function(module, exports) {',
-    this.runtime.source + ';',
-    '  return module.exports ||',
-    '      exports.' + exportName + ';',
-    '}({}, {});',
-    '/// -------------------------------------',
-    '/// --------- BEM-XJST Runtime End ------',
-    '/// -------------------------------------',
-    '',
-    'var api = new ' + engine + '(' + JSON.stringify(options) + ');',
-    '/// -------------------------------------',
-    '/// ------ BEM-XJST User-code Start -----',
-    '/// -------------------------------------',
-    'api.compile(function(' + locals.join(', ') + ') {',
-    code + ';',
-    '});',
-    'api.exportApply(exports);',
-    '/// -------------------------------------',
-    '/// ------ BEM-XJST User-code End -------',
-    '/// -------------------------------------\n'
-  ].join('\n');
+  var file = new SourceMap('bundle.js', { sourceMap: options.sourceMap });
```

Comment:
Let us make options in postcss: https://github.com/postcss/postcss/blob/master/docs/source-maps.md.

```js
{
    from: 'app.sass.css',
    to:   'app.css',
    map: { inline: false }
}
```